open_optical_gating/cli/prospective_optical_gating.py

```diff
@@ -257,6 +302,7 @@ def predict_trigger_wait(
 
     # Fixes sync error due to targetSyncPhase being 2pi greater than target phase
     # TODO check with JTs system
+    # TODO: JT writes: I have no idea what the above comment is about. Can this be clarified, or investigated?
```

Comment:
This comment must be three years old when I was still writing from your ObjC. I will check but it can probably be deleted now.